src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -634,7 +635,15 @@ export class HaSceneEditor extends SubscribeMixin(
       if (!entityReg.device_id) {
         continue;
       }
-      if (!this._devices.includes(entityReg.device_id)) {
+      const entity = config.entities[entityReg.entity_id];
+      const entityMetaData = config.metadata?.[entityReg.entity_id];
+      if (
+        !this._devices.includes(entityReg.device_id) &&
+        !(typeof entity === "string") &&
```

Comment:
Why check this?

Reply:
```suggestion
        typeof entity !== "string" &&
```

Reply:
This was added when the `entity_only` flag was set in the `entities` map and is indeed no longer needed 👍